enhancements/network/user-defined-network-segmentation.md

```diff
@@ -379,11 +382,106 @@ any user-defined primary network.
+| Topology       | The topological configuration for the network. Must be one of `layer2` or `layer3`.                                                                                                                                                                                                                                                                                                                | No       |
+| Role           | Select the network role in the pod, either `primary` or `secondary`.                                                                                                                                                                                                                                                                                                                               | No       |
+| MTU            | The maximum transmission unit (MTU).<br/>The default is 1400.                                                                                                                                                                                                                                                          | Yes      |
+| Subnets        | The subnet to use for the network across the cluster.<br/>E.g. 10.100.200.0/24.<br/>IPv6 (2001:DBB::/64) and dual-stack (192.168.100.0/24,2001:DBB::/64) subnets are supported.<br/>When omitted, the logical switch implementing the network only provides layer 2 communication, and users must configure IP addresses.<br/>Port security only prevents MAC spoofing if the subnets are omitted. | Yes      |
+| ExcludeSubnets | List of CIDRs and IP addresses.<br/>IP addresses are removed from the assignable IP address pool and are never passed to the pods.                                                                                                                                                                                                                                                                 | Yes      |
+
+The cluster scoped CRD should have the following additional field:
+
+| Field name | Description                                                                                           | optional |
+|------------|-------------------------------------------------------------------------------------------------------|----------|
+| Selector   | Key-value list of labels used as a selector for which namespace the network should be available for.  | No       |
+
+Since `localnet` topology for primary networks is non-goal, the CRDs controller should reject such requests.
```

Comment:
We are rejecting configuring secondary localnets ? I think what we should reject is configuring localnet as primary.

Reply:
Yes this is what I tired to express, will fix the text to clarify it and also mention it in validation section.